JustFakeIt/packages.config

```diff
@@ -1,11 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="FluentAssertions" version="4.0.0" targetFramework="net45" />
+  <package id="Microsoft.AspNet.Razor" version="3.0.0" targetFramework="net45" />
   <package id="Microsoft.Owin" version="3.0.1" targetFramework="net45" />
   <package id="Microsoft.Owin.Diagnostics" version="3.0.1" targetFramework="net45" />
   <package id="Microsoft.Owin.Host.HttpListener" version="3.0.1" targetFramework="net45" />
   <package id="Microsoft.Owin.Hosting" version="3.0.1" targetFramework="net45" />
   <package id="Microsoft.Owin.SelfHost" version="3.0.1" targetFramework="net45" />
   <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net45" />
   <package id="Owin" version="1.0" targetFramework="net45" />
+  <package id="RazorEngine" version="3.9.1" targetFramework="net45" />
```

Comment:
@slang25 sounds good, long term it might be good to pull the templating out into a separate package.